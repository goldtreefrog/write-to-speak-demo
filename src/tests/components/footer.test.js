import React from "react";
import ReactDOM from "react-dom";
import { shallow } from "enzyme";
import Footer from "./../../components/footer.js";

it("renders without crashing and matches snapshot", () => {
  const wrapper = shallow(<Footer />);
  expect(wrapper).toMatchSnapshot();
});
