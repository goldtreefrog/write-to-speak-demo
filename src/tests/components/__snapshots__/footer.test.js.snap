// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing and matches snapshot 1`] = `
<Aux>
  <footer
    role="contentinfo"
  >
    <p
      className="rv"
    >
      <a
        href="https://responsivevoice.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        ResponsiveVoice-NonCommercial
      </a>
       
      licensed under
       
      <a
        href="https://creativecommons.org/licenses/by-nc-nd/4.0/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="95x15"
          height="15"
          src="https://responsivevoice.org/wp-content/uploads/2014/08/95x15.png"
          title="ResponsiveVoice Text To Speech"
          width="95"
        />
         
      </a>
      (
      <a
        href="https://creativecommons.org/licenses/by-nc-nd/4.0/"
        rel="noopener noreferrer"
        target="_blank"
      >
        Creative Commons
      </a>
      )
    </p>
    <p>
      Website copyright Â©2018
      <span>
         
      </span>
       
      by Margaret Blauvelt
    </p>
  </footer>
</Aux>
`;
